index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="assets/css/styles.css">
+    <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    <script src="https://use.fontawesome.com/4eb9d85bb2.js"></script>
+</head>
+<body>
+    <header class="portfolio-header">
+        <h1>Jon D-ev</h1>
+        <nav>
+            <ul>
+                <li>
+                    <img src="assets/img/Union.png" alt="menu icon">
+                </li>
+            </ul>
+        </nav>
+    </header>
+    <article class="intro-wrapper">
```

Comment:
- [ ] Instead of using `<article>` tag to wrap this section, I'll suggest you use the `<section>` tag to wrap the entire headline section.